tools/zetasql-helper/zetasql-toolkit-core/src/main/java/com/google/zetasql/toolkit/validation/CannotRecreateExistingTable.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2023 Google LLC All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.zetasql.toolkit.validation;
+
+import com.google.common.collect.ImmutableList;
+import com.google.zetasql.NotFoundException;
+import com.google.zetasql.SimpleCatalog;
+import com.google.zetasql.resolvedast.ResolvedCreateStatementEnums;
+import com.google.zetasql.resolvedast.ResolvedNodes.ResolvedCreateTableAsSelectStmt;
+import com.google.zetasql.resolvedast.ResolvedNodes.ResolvedCreateTableStmt;
+import com.google.zetasql.resolvedast.ResolvedNodes.ResolvedCreateTableStmtBase;
+
+public class CannotRecreateExistingTable extends ValidatingVisitor {
```

Comment:
Since this class is a visitor (i.e. extends ResolvedNodes.Visitor), would it not be more comprehensible if it where called `CannotRecreateExistingTableVisitor`. 

All I am saying is that we add Visitor to the name since the class is a visitor

Reply:
Done in https://github.com/GoogleCloudPlatform/professional-services/pull/1005/commits/121e16b2924af0d31fb1c4e4f88f402d3378072c